src/components/DocumentationTopic.vue

```diff
@@ -520,6 +539,13 @@ export default {
     declarations({ primaryContentSections = [] }) {
       return primaryContentSections.filter(({ kind }) => kind === SectionKind.declarations);
     },
+    hasOtherDeclarations({ declarations = [] }) {
+      // there's always only 1 `declaration` at this level
+      return declarations.length ? declarations[0].declarations.some(decl => Object.prototype.hasOwnProperty.call(decl, 'otherDeclarations')) : false;
```

Comment:
Since you already have a default empty array, I'm not sure you need the ternary expression here.

Reply:
Accessing `declarations[0].declarations` would be an error.